examples/catalog/bin/sample_page.dart

```diff
@@ -61,7 +68,8 @@ String expandTemplate(String template, Map<String, String> values) {
   });
 }
 
-void writeExpandedTemplate(File output, String template, Map<String, String> values) {
+void writeExpandedTemplate(
+    File output, String template, Map<String, String> values) {
```

Comment:
Adding a line break here is not an improvement

Reply:
would you be happy with
```dart
void writeExpandedTemplate(
  File output,
  String template,
  Map<String, String> values,
) {
```
? 
The other option is allowing function declaration and to reach 100 or 120 chars before forcing wrapping.

Reply:
I think upping the limit is a good idea so long as developers can choose to write the multiline version. Not to put too fine a point on it, but I'd hope that the original version would be OK, and if the author had written this instead, the formatter would not collapse it to one line.
```
void writeExpandedTemplate(
  File output,
  String template,
  Map<String, String> values,
) {
```